tests/devices/test_default_qubit_autograd.py

```diff
@@ -233,7 +233,7 @@ def circuit(a, b):
         b = np.array(0.12, requires_grad=True)
 
         def cost(a, b):
-            prob_wire_1 = circuit(a, b)[0]
+            prob_wire_1 = circuit(a, b)

```

Comment:
Hmm, this implies that we have now one fewer dimensions, why is that?

Reply:
In non-tape mode, `BaseQNode` would always squeeze the QNode output, but `PassthruQNode` wouldn't. So if using `diff_method="backprop"`, the output would always have an extra dimension compared to parameter-shift.

This is no longer the case in tape mode in part due to `qml.math.squeeze` :D